app/models/blacklight_config/search_field_configuration.rb

```diff
@@ -27,7 +27,7 @@ def add_field(field)
     end
 
     def all_field
-      SearchField.new("all_fields", :qf => "all_text_timv")
+      SearchField.new("all_fields", :qf => "title_tesim description_tesim date_tesim all_text_timv")
```

Comment:
@terrellt it seems like when using all_text_timv only, other fields are not searchable. I wonder if we have to explicitly state which fields are copied to all_text_timv.